SlimApp/HomeView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ContentView.swift
+//  SlimApp
+//
+//  Created by Aleksandr Shapovalov on 10/01/23.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    var body: some View {
+        ZStack {
+            Color("Background")
+                .ignoresSafeArea()
+            NavigationView {
```

Comment:
Maybe consider fixing this ZStack/NavigationView order

Reply:
fixed on another branch with ClientList